dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```diff
@@ -242,6 +246,6 @@ CMD ["/app/apm.sh"]
 RUN echo -e '#!/bin/sh\nexec /usr/local/bin/docker-entrypoint' > /app/apm.sh && \
     chmod 0555 /app/apm.sh
 {{- else }}
-ENTRYPOINT ["/usr/bin/tini", "--", "/usr/local/bin/docker-entrypoint"]
+ENTRYPOINT ["/usr/bin/tini", "-s", "--", "/usr/local/bin/docker-entrypoint"]
```

Comment:
Does this change anything when `tini` is executed as PID 1?

Notice that this breaks compatibility with Linux < 3.4. But I guess we don't support any distribution with such old versions, so I guess this is fine.